framework/glsl_compiler.cpp

```diff
@@ -51,14 +51,17 @@ inline EShLanguage FindShaderLanguage(VkShaderStageFlagBits stage)
 		case VK_SHADER_STAGE_COMPUTE_BIT:
 			return EShLangCompute;
 
-		case VK_SHADER_STAGE_RAYGEN_BIT_NV:
-			return EShLangRayGenNV;
+		case VK_SHADER_STAGE_RAYGEN_BIT_KHR:
+			return EShLangRayGen;
 
-		case VK_SHADER_STAGE_MISS_BIT_NV:
-			return EShLangMissNV;
+		case VK_SHADER_STAGE_MISS_BIT_KHR:
+			return EShLangMiss;
 
-		case VK_SHADER_STAGE_CLOSEST_HIT_BIT_NV:
-			return EShLangClosestHitNV;
+		case VK_SHADER_STAGE_CLOSEST_HIT_BIT_KHR:
+			return EShLangClosestHit;
+
+		case VK_SHADER_STAGE_CALLABLE_BIT_KHR:
+			return EShLangCallable;
 
```

Comment:
For completion, there should be also the case for VK_SHADER_STAGE_INTERSECTION_BIT_KHR

Reply:
Good point. I have added all missing ray tracing shader stages to the PR, including any hit and intersection stages.